Sources/GRPC/CallHandlers/ClientStreamingCallHandler.swift

```diff
@@ -71,7 +74,7 @@ public class ClientStreamingCallHandler<RequestMessage: Message, ResponseMessage
     }
   }
 
-  public override func handlerAdded(context: ChannelHandlerContext) {
+  override public func handlerAdded(context: ChannelHandlerContext) {
```

Comment:
Why was this order swapped, but not elsewhere? (I prefer visibility first, BTW, unless it violates a style guide.)